README.md

```diff
@@ -69,21 +69,21 @@ An important note is that a failing test will raise an error, interrupting the b
 
 #### The `lspecIO` function
 
-`lspecIO` is meant to be used in files to be compiled and integrated in a testing infrastructure, as shown soon.
+`lspecIO` is meant to be used in files to be compiled and integrated in a testing infrastructure, as shown below.
 
 ```lean
-def fourIO : IO Nat :=
-  return 4
-
-def fiveIO : IO Nat :=
-  return 5
-
-def main := do
-  let four ← fourIO
-  let five ← fiveIO
-  lspecIO $
-    test "fourIO equals 4" (four = 4) $
-    test "fiveIO equals 5" (five = 5)
+def aaSuite := [
+    test "four equals four" (4 = 4)
+  ]
+
+def bbSuite := [
+    test "five equals five" (5 = 5)
+  ]
+
+def main := lspecIO $ .ofList [
+    ("aa", aaSuite),
+    ("bb", bbSuite)
+  ]
```

Comment:
To make usage abundantly clear, could we add a line of docs saying that you can then run the LSpec tests via `lake test` or `lake test <a|aa>` when including the following in `lakefile.lean`?
```
@[test_driver]
lean_exe Tests.Main
```
Even better would be a simple Lake project in `Examples/` to show the intended directory structure.